### PR TITLE
feat(organization): show quota usage, max, and refresh time

### PR DESCRIPTION
**Pull request by jdalton**
## Summary
- `socket organization quota` was hidden and only printed remaining units, so users had no way to see how close they were to their cap. The SDK already returns `maxQuota` and `nextWindowRefresh` on the same call — we now render all three.
- Unhide the `quota` subcommand so it shows up in help.
- Show `remaining / max (N% used)` plus a human-readable refresh window in text and markdown output.
- Fall back to remaining-only when `maxQuota` is missing so the command still works against older/partial responses.

## Test plan
- [x] `pnpm --filter @socketsecurity/cli run test:unit test/unit/commands/organization/` — 196 passed (two test files updated, one new test case added for maxQuota-missing fallback and one for refresh-time rendering)
- [x] `pnpm run type` — passes
- [x] `pnpm run lint` — passes

## Follow-ups (not in this PR)
- Preflight warning inside `handleApiCall` when a command's known quota cost exceeds the remaining budget. Would need caching to avoid an extra `getQuota` per invocation — worth a separate discussion.

---

> [!NOTE]
> **Low Risk**
> Low risk: changes are limited to CLI command discoverability and output formatting, with added tests to cover new rendering and edge cases.
> 
> **Overview**
> Makes `socket organization quota` visible in CLI help (unhides the subcommand) and updates its description.
> 
> Enhances quota output in text/markdown to include **remaining/max quota**, **% used**, and a **human-readable next refresh time**, with a fallback to remaining-only when `maxQuota` is missing; JSON output is unchanged. Updates and adds unit tests to cover the new formatting and refresh-time edge cases, and documents the behavior in `CHANGELOG.md`.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit b5cc26b80e715c49c752efce21af720669e22e81. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: packages/cli/src/commands/organization/output-quota.mts [hunk 1]
```diff
@@ -8,8 +8,47 @@ import type { CResult, OutputKind } from '../../types.mts'
...
+  }
+  const mins = Math.round(diffMs / 60_000)
+  if (mins < 60) {
+    return `${date} (in ${mins} min)`
+  }
+  const hours = Math.round(mins / 60)
+  if (hours < 48) {
+    return `${date} (in ${hours} h)`
+  }
+  const days = Math.round(hours / 24)
```
[the remark this answers is not in the document]
Comment (jdalton) [reply packages/cli/src/commands/organization/output-quota.mts]: Fixed in commit 3fb354587: each unit (min/h/d) is now computed directly from the raw `diffMs` delta, so the 89.5-min boundary and similar cascades now round correctly.

**Comment by jdalton** [comment]
@cursor review

File: packages/cli/src/commands/organization/output-quota.mts [hunk 2]
```diff
@@ -8,8 +8,47 @@ import type { CResult, OutputKind } from '../../types.mts'
...
+  const diffMs = ts - now
+  const date = new Date(ts).toISOString()
+  if (diffMs <= 0) {
+    return `${date} (due now)`
+  }
+  // Compute each unit directly from diffMs to avoid cascaded rounding
+  // errors — e.g. 89.5 min would round to 90 min, then to 2 h via the
+  // chain, even though 89.5 min is closer to 1 h.
+  if (diffMs < 3_600_000) {
+    return `${date} (in ${Math.round(diffMs / 60_000)} min)`
```
[the remark this answers is not in the document]
Comment (jdalton) [reply packages/cli/src/commands/organization/output-quota.mts:39]: Fixed in commit b5cc26b80. Under 60s we now display `<1 min` so we never round down to `0 min`. Promotion thresholds moved to 3_570_000 ms (59.5 min) and 171_000_000 ms (47.5 h) so rounding never lands on `60 min` or `48 h`. Added regression tests for both boundaries.

**Comment by jdalton** [comment]
@cursor review

**Comment by jdalton** [comment]
@cursor review

**Comment by jdalton** [comment]
@cursor review

**Comment by jdalton** [comment]
@cursor review

**Comment by jdalton** [comment]
@cursor review

**Review by billxinli — APPROVED** [review]
